loopfib.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+/*print fibonacci numbers with loop
+this one doesn't quite work and just prints out 23 when you type 4, I'm not quite sure why...
+  @author Aly Milich
+  */
+
+int main(){
+	int input;
+	int fibonacci[input];
```

Comment:
So this line causes a seg fault! this is because you aren't properly allocating memory? How do you do this? you move this statement after scanf("%d",&input); so that it properly allocates the memory